downstream/modules/platform/con-aap-backup-recommendations.adoc

```diff
@@ -0,0 +1,11 @@
+[id="aap-backup-recommendations"]
+
+= Backup recommendations
+
+
+[role="_abstract"]
+Recovering from data loss requires that you plan for and create backup resources of your {PlatformName} deployments on a regular basis. At a minimum, Red Hat recommends backing up deployments of {PlatformName} under the following circumstances:
+
+* Before upgrading your {PlatformName} deployments
+* Before upgrading your Openshift cluster
```

Comment:
Just a question/thought: Do we use the attribute {OCP} interchangeably with OpenShift? Just wondering about consistency. Wondering if we should have an attribute for just OpenShift. Just a thought since I see OpenShift a few times in addition to the OCPShort attribute.

Reply:
> Just a question/thought: Do we use the attribute {OCP} interchangeably with OpenShift? Just wondering about consistency. Wondering if we should have an attribute for just OpenShift. Just a thought since I see OpenShift a few times in addition to the OCPShort attribute.